lua/vscode/theme.lua

```diff
@@ -86,6 +86,9 @@ theme.set_highlights = function(opts)
     hl(0, 'SpellRare', { fg = c.vscRed, bg = c.vscBack, undercurl = true, sp = c.vscRed })
     hl(0, 'SpellLocal', { fg = c.vscRed, bg = c.vscBack, undercurl = true, sp = c.vscRed })
     hl(0, 'Whitespace', { fg = isDark and c.vscLineNumber or c.vscTabOther })
+    hl(0, 'NormalFloat', { bg = c.vscBack })
```

Comment:
I think `NormalFloat` should not be the same as `Normal` we need a different color for that. You can use whatever VS Code floating window or hover doc uses

Reply:
Good point! I changed the background to `c.vscPopupBack`, which should be good.

I didn't update the screenshot, though, but I would need to rewrite the Neovim 0.10 overlay for my Nix :grimacing: